controllers/usersController.js

```diff
@@ -3,14 +3,14 @@ const { getAllUsers, getUserById } = require('../queries/usersQueries');
 
 const usersController = express.Router();
 
-usersController.get('/', (req, res) => {
-  users = getAllUsers() || [];
+usersController.get('/', async (req, res) => {
+  users = await getAllUsers() || [];
```

Comment:
This is my fault not yours but wanted to point it out. There should be a `const` or `let` before `users`. The original repo could should have had this!